plugins/postcss-rebase-url/package.json

```diff
@@ -0,0 +1,85 @@
+{
+	"name": "@csstools/postcss-rebase-url",
+	"description": "Rebase url() functions when transforming CSS",
+	"version": "0.1.0-alpha.0",
```

Comment:
Is this intended? Not really against it but just double checking since we don't usually go this path.

Reply:
I published an alpha version so that I could verify everything in : https://github.com/romainmenke/css-import-tests